shell/platform/android/io/flutter/app/FlutterPluginRegistry.java

```diff
@@ -30,10 +30,12 @@
              PluginRegistry.ViewDestroyListener {
     private static final String TAG = "FlutterPluginRegistry";
 
+    public interface ViewRegistry extends BinaryMessenger, TextureRegistry {}
```

Comment:
I don't think this interface is buying us anything.  I think you can use FlutterView exactly as before and you get the same result.

Reply:
You don't exactly get the same result. TextureView can no longer setup a registry (i.e. call `attach`) if we leave this as is.